### ISSUE TITLE
SPE-Remoting does not work when Windows Authentication is enabled on Sitecore instance

### ISSUE THREAD

**Issue by pbering** [SitecorePowerShell/Console#379]
**Feature Request**

New-ScriptSession should have an optional -Credential parameter of type PSCredential so we can parse on Windows credentials from Get-Credential. I see you use New-WebServiceProxy which already supports PSCredentials.

**Comment by michaellwest** [comment]
I'll take a look and see what I can do. Should be pretty straight forward just as you said.

**Comment by michaellwest** [comment]
Made some changes to the commands. Once I test those on a computer joined to a domain I will post the update.

**Comment by michaellwest** [comment]
Implemented. 

I tested this out at work and even made some improvements. The proxy object is now returned as part of `New-ScriptSession`.  I ran a simple test for 10 consecutive calls and the time went from 1.5 sec to .5 sec. Will package this up soon for you to test our yourself.

@pbering Is the use of *UseDefaultCredentials* important to you? If not I'll keep it simple and leave that out for now.
